### PR TITLE
Adds the Ti'Rakqi Freighter

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds a skrell themed third party ship at the request of the lore team. Most of the mapping was done by butterrobber

### CODE REVIEW

**Review by TheDanseMacabre — CHANGES_REQUESTED** [review]
I would recommend removing some of the guns present. This ship seems over-armed relative to the rest of the third party ships, which tend to have a highly limited armament across the board. I would also recommend removing the pirate coat in the cockpit, and the meme very rare hat built into the wall. Generally speaking, I find these "easter eggs" to be a bad design decision.

**Comment by DreamySkrell** [comment]
IMO: The layout is really weird. Both like, subjectively, but also "objectively".

The small shuttle can't leave through the hangar doors.

All the rooms are huge, and mostly empty.

They are pirates, but there's no brig to hold any prisoners or etc.

There's no dedicated EVA gear room. Airlock is from outside straight to that warehouse.

**Comment by Butterrobber202** [comment]
> IMO: The layout is really weird. Both like, subjectively, but also "objectively".
> 
> The small shuttle can't leave through the hangar doors.
> 
> All the rooms are huge, and mostly empty.
> 
> They are pirates, but there's no brig to hold any prisoners or etc.
> 
> There's no dedicated EVA gear room. Airlock is from outside straight to that warehouse.
> 
> Am I blind? Aren't those guns energy/laser? I see no rechargers.

Small Shuttle can leave.
Rooms have extra space so the crew make can their own modifications to the ship as they please.
This ship is designed for smuggling. (Also, hostage RP is about as fun as smashing your foot with a rock.)
Overkill to have a room that's only purpose is EVA. The Engineer room has the equipment needed.
As for the rechargers. I may have just forgot them.

**Comment by DreamySkrell** [comment]
Talked this over in the discord. I have other concerns, but, ehhh.

**Review by listerla — APPROVED** [review]

File: maps/away/ships/tirakqi_freighter_ghostroles.dm [hunk 1]
```diff
@@ -0,0 +1,176 @@
+/datum/ghostspawner/human/tirakqi_crew
+	short_name = "tirakqi_crew"
+	name = "Ti'Rakqi Qu'fup"
+	desc = "You crew the ship, mop the floors, cook the meals, and shoot whoever gets to close too the goods. Try to show some initiative!"
```
Comment (Arrow768) [inline maps/away/ships/tirakqi_freighter_ghostroles.dm:4]:
```suggestion
	desc = "You crew the ship, mop the floors, cook the meals, and shoot whoever gets too close to the goods. Try to show some initiative!"
```

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**